x/campaign/keeper/grpc_auctions_by_campaign.go

```diff
@@ -0,0 +1,38 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	fundraisingtypes "github.com/tendermint/fundraising/x/fundraising/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/tendermint/spn/x/campaign/types"
+)
+
+func (k Keeper) AuctionsOfCampaign(goCtx context.Context, req *types.QueryAuctionsOfCampaignRequest) (*types.QueryAuctionsOfCampaignResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
+
```

Comment:
I think we should return an error not found if the campaign doesn't exist

Reply:
I did this on purpose. It should just return an empty list as of now, right?
Why the error should be better? It's a query, not a tx. You would know if the campaign exist already by querying for the campaign itself.

Reply:
That's a small optimization, you would not iterate auctions if the campaign doesn't exist and therefore no auction would be found